### PR TITLE
Document client OS support in a table

### PR DESCRIPTION
**Pull request by ptman**
Would this be a good documentation improvement as discussed in #58 ?

### CODE REVIEW

**Comment by qbit** [comment]
Needs a line for OpenBSD! :D

**Comment by ptman** [comment]
I just looked at https://tailscale.com/download for a list

File: README.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ Headscale implements this coordination server.
 - [x] Share nodes between ~~users~~ namespaces
 - [ ] MagicDNS / Smart DNS
 
+## Client OS support
+
+| OS | Supports headscale |
+| --- | --- |
+| Linux | Yes |
+| macOS | Yes |
```
Comment (kradalby) [inline README.md:38]:
```suggestion
| macOS | Yes (see `/apple` on your headscale for more information) |
```

**Comment by kradalby** [comment]
I think this looks like a good start :), I added some comments to point users in the right direction.

Without having tested, I assume Windows users can override the control URL via som regedit/HKEY magic, would be great if someone who figure that out can add a note about that :)

**Review by kradalby — APPROVED** [review]

**Pull request merged**